### DISCUSSION TITLE
Changing the turbulence intensity

### DISCUSSION THREAD

**Discussion by AntonioCervantes** [xcompact3d/Incompact3d#157] [Q&A] [answered]
Hello all,
I am trying to run 3 LES channel flow simulations with turbulence intensities of 1%, 3%, and 5%, but cannot figure out how to do that. I am using the Sandbox flow configuration with an initial velocity field setup using the xcompact3d-toolbox. I've tried to change the `init_noise` and `inflow_noise` variables in the input files but when I calculated the turbulence intensity from the simulation data, I found each one to be the same values of around 5-6%. The simulation boundary conditions I'm using are periodic inlet/outlet, periodic side walls, and no slip floor. I've created my own fully developed power law boundary layer using the xcompact3d toolbox package. Is there any way to change and control the turbulence intensity in the channel for a power law boundary condition? I appreciate any input or guidance that you can provide.

**Comment by mathrack** [comment]
Hello,

I am afraid I do not really understand your question. If you are running a turbulent channel flow with periodicity in the stream-wise and span-wise directions, then the only boundary condition is the no-slip boundary condition at the top and bottom walls.

If you want to simulate a turbulent channel flow, it is probably wiser to use [one of the provided examples](https://github.com/xcompact3d/Incompact3d/blob/master/examples/Channel-Flow/input_DNS_Re180_LR_explicittime.i3d).

If you want to simulate some boundary layers, you might want to use one of the provided examples : https://github.com/xcompact3d/Incompact3d/tree/master/examples/ABL or https://github.com/xcompact3d/Incompact3d/tree/master/examples/Turbulent-Boundary-Layer

For sandbox-specific questions, please refer to https://github.com/fschuch/xcompact3d_toolbox/

If you want to understand how the parameters `init_noise` or `inflow_noise` are used in the code, please look at the source code. For instance, the command `grep -i inflow_noise` leads to : 
```
BC-Uniform.f90:        bxx1(j,k)=bxx1(j,k)+(two*bxo(j,k)-one)*inflow_noise*um
BC-Uniform.f90:        bxy1(j,k)=bxy1(j,k)+(two*byo(j,k)-one)*inflow_noise*um
BC-Uniform.f90:        bxz1(j,k)=bxz1(j,k)+(two*bzo(j,k)-one)*inflow_noise*um
BC-Cylinder.f90:          bxx1(j,k)=u1+bxo(j,k)*inflow_noise
BC-Cylinder.f90:          bxy1(j,k)=zero+byo(j,k)*inflow_noise
BC-Cylinder.f90:          bxz1(j,k)=zero+bzo(j,k)*inflow_noise
BC-ABL.f90:        bxx1(j,k)=bxx1(j,k)+(two*bxo(j,k)-one)*inflow_noise*um
BC-ABL.f90:        bxy1(j,k)=bxy1(j,k)+(two*byo(j,k)-one)*inflow_noise*um
BC-ABL.f90:        bxz1(j,k)=bxz1(j,k)+(two*bzo(j,k)-one)*inflow_noise*um
parameters.f90:       itype, iin, re, u1, u2, init_noise, inflow_noise, &
parameters.f90:  inflow_noise = zero
BC-Sandbox.f90:    bxo = (two*bxo - one)*inflow_noise*noise_mod_x1
BC-Sandbox.f90:    byo = (two*byo - one)*inflow_noise*noise_mod_x1
BC-Sandbox.f90:    bzo = (two*bzo - one)*inflow_noise*noise_mod_x1
BC-Sandbox.f90:    bxo = (two*bxo - one)*inflow_noise*noise_mod_x1
BC-Sandbox.f90:    byo = (two*byo - one)*inflow_noise*noise_mod_x1
BC-Sandbox.f90:    bzo = (two*bzo - one)*inflow_noise*noise_mod_x1
module_param.f90:  real(mytype) :: dt,re,xnu,init_noise,inflow_noise,u1,u2,angle,anglex,angley
```

**Comment by slaizet** [comment, marked as answer]
Hi,
To complement Cedric's answer, you can change the turbulence intensity by simply changing the Reynolds number. `inflow_noise` is not used with periodic BC and `init_noise` is only for the initial condition.
Best,
Sylvain

**Reply by AntonioCervantes** [reply]
Hello,

Thank you for your response. I've tested the channel flow example linked by Cedric with different Reynolds numbers and found the turbulence intensity was changed by doing that. An additional question I have is how can I model a channel flow where I can create my own power law velocity profile?

**Reply by mathrack** [reply]
You might want to simulate a turbulent boundary layer and impose a power law velocity profile at the inlet ? Or a periodic channel flow with source terms to represent this power law profile ?

This answer is highly speculative. You have nowhere clarified the problem at hand.

When investigating turbulent flows, it is not usual to "create" velocity profiles.
